crates/bevy_gilrs/src/lib.rs

```diff
@@ -1,4 +1,10 @@
+//! Systems and type definitions for gamepad handling in Bevy.
+//!
+//! This crate is built on top of [GilRs](https://docs.rs/gilrs/latest/gilrs), a library
```

Comment:
I wonder if using `[GilRs](gilrs)` would cause docs.rs to link to the right version?

Reply:
Seems like this works properly locally (if the dep doc is built of course), and on docs.rs! ([example I found](https://docs.rs/test8981/0.1.0/test8981/))